longruntests/longruntests_suite_test.go

```diff
@@ -0,0 +1,152 @@
+package longruntests_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/deis/workflow-e2e/utils"
```

Comment:
I had used it since they are already used in the repo. I am happy to change them but just wanted to know the reason for not using the anonymous  import.